tests/pdf-viewer/pdf-viewer-search-panel.html

```diff
@@ -0,0 +1,82 @@
+<head>
+    <title>PDF Viewer With Search Panel</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" data-role="kendo-theme" href="/packages/default/dist/all.css">
+    <link rel="stylesheet" href="/packages/html/assets/styles.css">
+    <script src="/packages/html/assets/scripts.js"></script>
+</head>
+<body id="app" class="k-body">
+    <style>
+        #test-area .k-pdf-viewer {
+            height: 600px;
+        }
+
+    </style>
+    <div id="test-area" class="k-d-grid k-grid-cols-1">
+        <div class="k-widget k-pdf-viewer">
+            <div class="k-toolbar k-widget k-toolbar-resizable">
+                <div class="k-pager">
+                    <button class="k-pager-nav k-pager-first k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                        <span class="k-button-icon k-icon k-i-arrow-end-left"></span>
+                    </button>
+                    <button class="k-pager-nav k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                        <span class="k-button-icon k-icon k-i-arrow-60-left"></span>
+                    </button>
+                    <span class="k-pager-input k-label">
+                        <span class="k-textbox k-input k-input-md k-input-solid k-rounded-md">
+                            <input type="text" class="k-input-inner" placeholder="" autocomplete="off" value="">
+                        </span>of 4 pages
+                    </span>
+                    <button class="k-pager-nav k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                        <span class="k-button-icon k-icon k-i-arrow-60-right"></span>
+                    </button>
+                    <button class="k-pager-nav k-pager-last k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                        <span class="k-button-icon k-icon k-i-arrow-end-right"></span>
+                    </button>
+                </div>
+                <span class="k-spacer"></span>
+                <button class="k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                    <span class="k-button-icon k-icon k-i-file-pdf"></span>
+                </button>
+                <button class="k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                    <span class="k-button-icon k-icon k-i-print"></span>
+                </button>
+                <button class="k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                    <span class="k-button-icon k-icon k-i-fullscreen"></span>
+                </button>
+            </div>
+            <div class="k-canvas k-pdf-viewer-canvas k-pos-relative k-overflow-auto">
+                <div class="k-search-panel k-pos-absolute k-top-center">
```

Comment:
When the `k-pdf-viewer-canvas` element is scrolled (which is the case if you scroll the document or click on next search match), the search panel is no longer visible:
![pdf-viewer-search-scroll](https://user-images.githubusercontent.com/1857705/186669310-3dc2d513-4ec7-4b3c-a947-c4826076c57d.gif)

Reply:
On a smaller screen the matches words are somewhat misaligned:
![image](https://user-images.githubusercontent.com/1857705/186672659-9758ca62-9b7c-4f76-a86c-113d4bf07ed0.png)
seems like we can actually look ok even on this screen size, but the text is not stretched.

Another view
![image](https://user-images.githubusercontent.com/1857705/186672990-7fed7922-f1f9-4821-8df2-20278a96361b.png)

I am using http://192.168.100.4:8080/tests/pdf-viewer/pdf-viewer-search-panel.html for testing.